cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -124,26 +124,28 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
 
         imagebuilder.CfnComponent(
             self,
-            id="PClusterComponent",
-            name="-".join(["PCluster", resources_prefix]),
+            id="ParallelClusterComponent",
+            name="-".join(["ParallelCluster", resources_prefix]),
```

Comment:
same consideration for the name convention

Reply:
done